webservices/tasks/legal_docs.py

```diff
@@ -98,10 +98,15 @@ def refresh_cases(conn):
     rs = conn.execute(RECENTLY_MODIFIED_CASES)
     if rs.returns_rows:
         load_count = 0
+        deleted_case_count = 0
         for row in rs:
             logger.info("%s %s found modified at %s", row["case_type"], row["case_no"], row["pg_date"])
             load_cases(row["case_type"], row["case_no"])
-            load_count += 1
-        logger.info("Total of %d case(s) loaded", load_count)
+            if row["published_flg"]:
```

Comment:
@pkfec should we put the total logging statements outside the `for` loop like we did on line 105 before?

https://github.com/fecgov/openFEC/blob/7c7454ad2a4fe06f73d7eefabdca9c2f6b21d051/webservices/tasks/legal_docs.py#L102-L110

Reply:
@lbeaufort 

I tried to load AF 7283, and unpublish MUR's 7281, 7297 on my local `cfdm_test` db. Below are the log statements which show the count of case(s) that are loaded and unpublished.  

[2019-04-26 16:35:00,005: INFO/ForkPoolWorker-4] MUR 7281 found modified at 2019-04-27 15:50:00
[2019-04-26 16:35:00,005: INFO/ForkPoolWorker-4] Loading MUR(s)
[2019-04-26 16:35:00,069: INFO/ForkPoolWorker-4] Found an unpublished case - deleting from ES
[2019-04-26 16:35:00,074: INFO/ForkPoolWorker-4] Sucessfully deleted MUR 7281 from ES
**[2019-04-26 16:35:00,075: INFO/ForkPoolWorker-4] Total of 1 case(s) unpublished...**
[2019-04-26 16:35:00,075: INFO/ForkPoolWorker-4] AF 7283 found modified at 2019-04-27 15:50:00
[2019-04-26 16:35:00,075: INFO/ForkPoolWorker-4] Loading AF(s)
[2019-04-26 16:35:00,138: INFO/ForkPoolWorker-4] Loading AF: 7283
[2019-04-26 16:35:00,150: INFO/ForkPoolWorker-4] 1 AF(s) loaded
**[2019-04-26 16:35:00,151: INFO/ForkPoolWorker-4] Total of 1 case(s) loaded...**
[2019-04-26 16:35:00,151: INFO/ForkPoolWorker-4] MUR 7297 found modified at 2019-04-27 15:50:00
[2019-04-26 16:35:00,152: INFO/ForkPoolWorker-4] Loading MUR(s)
[2019-04-26 16:35:00,235: INFO/ForkPoolWorker-4] Found an unpublished case - deleting from ES
[2019-04-26 16:35:00,241: INFO/ForkPoolWorker-4] Sucessfully deleted MUR 7297 from ES
**[2019-04-26 16:35:00,241: INFO/ForkPoolWorker-4] Total of 2 case(s) unpublished...**